README.md

```diff
@@ -52,44 +62,85 @@ After you have cloned the repository:
 
 Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
 
-### Configure external API access keys
+#### Configuration
+
+The app’s configuration are following the [Twelve Factor](https://12factor.net/config)
+methodology, which means they are stored in the `environment`. You may use `source`,
+[`direnv`](https://direnv.net), [`nv`](https://github.com/jcouture/nv) or a custom script of your own.
+
+```properties
+DATABASE_URL="postgres://postgres:development@localhost/montreal_elixir_dev"
+
+TWITTER_CONSUMER_KEY=
+TWITTER_CONSUMER_SECRET=
+TWITTER_ACCESS_TOKEN=
+TWITTER_ACCESS_TOKEN_SECRET=
+
+YOUTUBE_API_KEY=
+```
```

Comment:
Shall we just have a link to the config file rather than [inlining it in the README](https://github.com/gcauchon/montreal-elixir-website/tree/feature/development-config-as-env#configuration)? I think this just adds visual noise and a chance this get out of sync with the actual file.

Reply:
We don’t want the config file itself in the codebase not to commit keys by mistake. I could link to a sample though, but that would assume everyone would use the same tool to inject environment variables. Since you are suggesting `heroku local` in another comment, I'll create a sample file in the format it expect and [link]() to it.

Reply:
Sorry, yes I was meaning that we should have a sample file.